src/main/java/edu/kis/powp/jobs2d/drivers/adapter/LineDrawerAdapter.java

```diff
@@ -0,0 +1,31 @@
+package edu.kis.powp.jobs2d.drivers.adapter;
+
+import edu.kis.legacy.drawer.panel.DrawPanelController;
+import edu.kis.legacy.drawer.shape.ILine;
+import edu.kis.legacy.drawer.shape.LineFactory;
+
+
+public class LineDrawerAdapter extends DrawerJob2dDriverAdapter {
+	public LineDrawerAdapter(DrawPanelController controller) {
+		super(controller);
+	}
+
+	@Override
+	public void operateTo(int x, int y) {
+		ILine line = LineFactory.getSpecialLine();
```

Comment:
Hardcoded line type. What about 3.3.2 test?
Direct coupling with LineFactory class is unnecessary.